ivy_tests/test_ivy/test_frontends/test_jax/test_activations.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+from hypothesis import given, strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+import ivy.functional.backends.jax as ivy_jax
+
+
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=ivy_jax.valid_float_dtypes),
```

Comment:
We should test for all data types so we should use `ivy_jax.valid_numeric_dtypes`